pages/icon/text-align.page.tsx

```diff
@@ -14,29 +14,29 @@ export default function IconScenario() {
       </ul>
       <ScreenshotArea>
         <Box fontSize="display-l" fontWeight="light">
-          <Icon name="settings" size="large" /> Display large <Icon name="external" size="inherit" />
+          <Icon name="gen-ai" size="large" /> Display large <Icon name="external" size="inherit" />
```

Comment:
Why does the other page make the icon configurable and here we change the default? This will break screenshot tests, is it expected?

Reply:
I was going to use this page for the screenshot tests, knowing that it would break screenshot tests on the first run. The other page was just to make it easier to see any icon in our dev environment.